### ISSUE TITLE
Getting errors, but data seems to be coming through from the API

### ISSUE THREAD

**Issue by Sjorsa** [adriankumpf/teslamate#3389]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I am not getting any data anymore into the database/grafana. Seems like the lat/lon is not coming through properly?

`[latitude: {"can't be blank", [validation: :required]}, longitude: {"can't be blank", [validation: :required]}]`

### Relevant log output

```shell
teslamate  | 2023-11-01 22:05:09.799 car_id=1 [info] Start / :online
teslamate  | 2023-11-01 22:05:09.801 [error] GenStateMachine :"1" terminating
teslamate  | ** (MatchError) no match of right hand side value: {:error, #Ecto.Changeset<action: :insert, changes: %{date: ~U[2023-11-01 21:05:09.686000Z], power: 0, odometer: 101369.761815, battery_heater: false, driver_temp_setting: Decimal.new("20.5"), fan_status: 0, inside_temp: Decimal.new("12.7"), is_climate_on: false, is_front_defroster_on: false, is_rear_defroster_on: false, outside_temp: Decimal.new("10.5"), passenger_temp_setting: Decimal.new("20.5"), tpms_pressure_fl: Decimal.new("2.7"), tpms_pressure_fr: Decimal.new("2.725"), tpms_pressure_rl: Decimal.new("2.7"), tpms_pressure_rr: Decimal.new("2.75"), battery_heater_on: false, battery_level: 56, usable_battery_level: 55, ideal_battery_range_km: Decimal.new("266.17"), rated_battery_range_km: Decimal.new("266.17"), est_battery_range_km: Decimal.new("214.64")}, errors: [latitude: {"can't be blank", [validation: :required]}, longitude: {"can't be blank", [validation: :required]}], data: #TeslaMate.Log.Position<>, valid?: false>}
teslamate  |     (teslamate 1.27.3) lib/teslamate/vehicles/vehicle.ex:721: anonymous fn/4 in TeslaMate.Vehicles.Vehicle.handle_event/4
teslamate  |     (ecto_sql 3.10.1) lib/ecto/adapters/sql.ex:1203: anonymous fn/3 in Ecto.Adapters.SQL.checkout_or_transaction/4
teslamate  |     (db_connection 2.5.0) lib/db_connection.ex:1630: DBConnection.run_transaction/4
teslamate  |     (teslamate 1.27.3) lib/teslamate/vehicles/vehicle.ex:713: TeslaMate.Vehicles.Vehicle.handle_event/4
teslamate  |     (stdlib 5.0.1) gen_statem.erl:1379: :gen_statem.loop_state_callback/11
teslamate  |     (stdlib 5.0.1) proc_lib.erl:241: :proc_lib.init_p_do_apply/3
teslamate  | 2023-11-01 22:05:09.839 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/xxxxx -> 429 (36.044 ms)
teslamate  | 2023-11-01 22:05:09.839 [warning] TeslaApi.Error / "Retry later\n"
teslamate  | 2023-11-01 22:05:09.839 car_id=1 [error] Error / :unknown
```

### Type of installation

Docker

### Version

1.27.3

**Comment by cwanja** [comment]
See #3371, #3370 and #3386. A "fix" is in #3370 and we are waiting to see if @adriankumpf pushes a formal fix to the TeslaMate.

**Comment by Sjorsa** [comment]
Woops, should've looked harder. I searched for `can't be blank validation required`, apparently github search isn't that thorough :P
